caddyhttp/httpserver/roller.go

```diff
@@ -54,17 +55,32 @@ func IsLogRollerSubdirective(subdir string) bool {
 		subdir == directiveRotateCompress
 }
 
+var invalidRollerParameterErr = errors.New("invalid roller parameter")
+
 // ParseRoller parses roller contents out of c.
-func ParseRoller(l *LogRoller, what string, where string) error {
+func ParseRoller(l *LogRoller, what string, where ...string) error {
 	if l == nil {
 		l = DefaultLogRoller()
 	}
-	var value int
-	var err error
-	value, err = strconv.Atoi(where)
-	if what != directiveRotateCompress && err != nil {
-		return err
+
+	// rotate_compress doesn't accept any parameters.
+	// others only accept one parameter
+	if (what == directiveRotateCompress && len(where) != 0) ||
```

Comment:
If anyone was using `rotate_compress` and put something after it to get it to work (like I did) this is a breaking change. Probably worth noting in release notes.